build/wasm.js

```diff
@@ -1,13 +1,14 @@
 'use strict'
 
+const WASM_BUILDER_CONTAINER = 'ghcr.io/nodejs/wasm-builder@sha256:975f391d907e42a75b8c72eb77c782181e941608687d4d8694c3e9df415a0970' // v0.0.9
```

Comment:
Are there plans to provide regular updates to it?

Reply:
I have been thinking a bit about that, I think we want to update regularly but not necessarily too often. At least a month before every new Node.js major but I could see more often making sense as well. 

The repo where it is maintained has release please and all of the automation to generate a new vesion so it's as simple as a PR to update the dockerfile used to build the container and once that lands the rest is pretty much automated.

@metcoder95 how often do you think an update would make sense ? How often was undici updating?

Reply:
I cannot develop a cadence, as I'd say that should be up to the things added to the new version. If it contains security fixes, maybe faster than usual, otherwise we can make a check every time we prepare a new `undici`'s release.

@mcollina @ronag wdyt?